### DISCUSSION TITLE
Edge line change on cutting diagram

### DISCUSSION THREAD

**Discussion by Andy34-86** [lairdubois/lairdubois-opencutlist-sketchup-extension#462] [Q&A]
Hello. Can I change line weight on cutting diagra.
![IMG_20230122_134735](https://user-images.githubusercontent.com/123301834/213914180-bcb9f85f-108c-40ad-a836-16d2ac3bd0ee.png)

**Comment by bbeaulant** [comment]
Hi Andy,

Currently not. 
You want to change the black ou blue one ? They are to big or too small ?

**Reply by Andy34-86** [reply]
Too big lines for edges , blue one.

**Comment by bbeaulant** [comment]
Too big for small parts ?

**Reply by Andy34-86** [reply]
Now blue lines are 3px , i want make them 2 or 1,5 px. Thanks.

**Reply by Andy34-86** [reply]
All blue lines.

**Reply by Andy34-86** [reply]
I fond .css file and change lines from blue to red , but cannot find file were i can change lines weight.

**Reply by bbeaulant** [reply]
This "width" defined in twig template not in CSS, because it is a constant of the SVG generator. 
To change it you have to edit the javascript compiled `twig-templates.js` file and search `edgeWidth`.

**Reply by Andy34-86** [reply]
Ok.Tkanks.

**Reply by Andy34-86** [reply]
This "width" defined in twig template not in CSS, because it is a constant of the SVG generator.
To change it you have to edit the javascript compiled twig-templates.js file and search edgeWidth.
Hello. Can answer witch line will be correct for edge weight changing. Thanks.

**Comment by Andy34-86** [comment]
This "width" defined in twig template not in CSS, because it is a constant of the SVG generator.
To change it you have to edit the javascript compiled twig-templates.js file and search edgeWidth.
Hello. Can answer witch line will be correct for edge weight changing. Thanks.

**Reply by Andy34-86** [reply]
Can answer witch line will be correct for edge weight changing. Thanks.

**Reply by Andy34-86** [reply]
Wich twig-template i need to open ???
![Twig](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/123301834/ae284900-40f8-4f41-bc0b-4962fd9150fe)

**Reply by bbeaulant** [reply]
The one that contains edgeWidth word ...
